lib/reporters/base.js

```diff
@@ -318,6 +326,13 @@ function Base(runner, options) {
   this.runner = runner;
   this.stats = runner.stats; // assigned so Reporters keep a closer reference
 
+  var maxDiffSizeOpt =
+    this.options.reporterOption && this.options.reporterOption.maxDiffSize;
+  console.error();
+  if (maxDiffSizeOpt !== undefined && !Number.isNaN(Number(maxDiffSizeOpt))) {
```

Comment:
The same here with `Number.isNaN()`  and IE11. You can replace it with the global `isNaN()`. Both functions ar weird, but the result should be the same in our case.